index.html

```diff
@@ -100,19 +102,20 @@ <h3 class="article__title">How can I help?</h3>
                     Learn more
                   </a>
                 </p>
-              </div>
-            </div>
+              </article>
+            </section>
           </div>
-        </div>
+        
+        </main>
 
-        <div class="sidebar">
+        <aside>
           <h2 class="heading-underline">Upcoming events</h2>
 
-          <div class="article">
+          <article>
             <img
               class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
-              alt=""
+              src="/images/spring-fundraisers_thumbnail.jpg"
```

Comment:
same thing
( https://github.com/CodeYourFuture/bikes-for-refugees/pull/334/files#r1003856170 )
applies here